src/components/Card/components/SbCardHeader.vue

```diff
@@ -0,0 +1,30 @@
+<template>
+  <div class="sb-card__header" v-bind="$attrs">
+    <component :is="as" v-if="title" class="sb-card__title"
+      >{{ title }}
+    </component>
+    <slot v-else></slot>
+    <SbCardOptions v-if="options" :options="options"></SbCardOptions>
```

Comment:
please, use self-close for components without content: https://v2.vuejs.org/v2/style-guide/?redirect=true#Self-closing-components-strongly-recommended

Reply:
@gustavomelki  i've self-closed the components without content